pkg/querier/frontend/queue_test.go

```diff
@@ -84,3 +89,36 @@ func TestDequeuesExpiredRequests(t *testing.T) {
 	_, ok = f.queues[userID2]
 	require.Equal(t, false, ok)
 }
+
+func BenchmarkGetNextRequest(b *testing.B) {
+	var config Config
+	flagext.DefaultValues(&config)
+	config.MaxOutstandingPerTenant = 2
+
+	const numTenants = 50
+
+	frontends := make([]*Frontend, 0, b.N)
+
+	for n := 0; n < b.N; n++ {
+		f, _ := setupFrontend(config)
```

Comment:
Maybe lets not ignore any errors here (or anywhere in benchmark)?

Reply:
Good point.  Resolved.